### PR TITLE
Fix undeclared variables in custom activity search

### PR DESCRIPTION
**Pull request by colemanw**

### CODE REVIEW

**Comment by joseltorres** [comment]
working on this

File: CRM/Contact/Form/Search/Custom/ActivitySearch.php [hunk 1]
```diff
@@ -204,7 +201,7 @@ public function all(
     }
 
     // add custom group fields to SELECT and FROM clause
-    $groupTree = CRM_Core_BAO_CustomGroup::getTree('Activity', $form, NULL, NULL, '', NULL);
+    $groupTree = CRM_Core_BAO_CustomGroup::getTree('Activity', $this);
```
Comment (joseltorres) [inline CRM/Contact/Form/Search/Custom/ActivitySearch.php]: The second argument is deprecated and should be removed.

**Comment by joseltorres** [comment]
Visited the custom search for activities and found notices on the live demo site. Tested the PR Jenkins build and all notices have been cleared up. In inspecting code we found an argument that has been deprecated. 

This PR should be made better by removing the deprecated argument, see comment above.

**Comment by colemanw** [comment]
@joseltorres thanks for the review. I've removed the extra param.

**Pull request merged**